parser.py

```diff
@@ -9,23 +9,52 @@
 import sys
 from os import listdir
 from os.path import isfile, join
-nltk.download('punkt')
+from collections import defaultdict
+import json
 
 def GetLanguageId(language):
-    language_id_list = logic.SQLQuery("select Lang_ID from lang_ref where Lang_Desc = '"+ language +"'")
+    language_id_list = logic.SQLQuery("select Lang_ID from lang_ref where Lang_Desc = '" + language + "'")
 
-    # Condition for language not present in database
     if len(language_id_list) == 0:
         return -1
     else:
         return language_id_list[0]
 
+def StoreIndexing(dictionary):
+    f = open("indexing.txt", "w")
+    f.write(dictionary)
+    f.close()
+
+def CreateIndexing(sentence, line_id, dictionary):
+    word_list = sentence.split()
+    # number of parts formed for each tagged word in the corpus(word/pos)
+    tagged_word_parts = 2   
+    
+    for word in word_list:
+        # skip untagged words
```

Comment:
I like this idea, when we will be using untagged corpus we can simply change this.